### ISSUE TITLE
[DEP0111] DeprecationWarning: Access to process.binding('http_parser') is deprecated.

### ISSUE THREAD

**Issue by meesha81** [creationix/http-parser-js#84]
I would like to ask if it would be possible to adjust the use depending on the published DEP0111.

More info here:
https://nodejs.org/api/deprecations.html#DEP0111

Node currently do not want this usage:
process.binding('http_parser').HTTPParser = require('http-parser-js').HTTPParser;

**Comment by Jimbly** [comment]
Nope, if you want to change how Node's internal http parsing behaves by replacing it with this module, you need to access the node internal bindings, no way around that :(.

You can maybe run `node --expose-internals` and use `const { internalBinding } = require('internal/test/binding'); internalBinding('http_parser')` instead, or something like that, as that's what's currently used in [the Node.js test that helps ensure this module will keep working](https://github.com/nodejs/node/blob/e46c680bf2b211bbd52cf959ca17ee98c7f657f5/test/parallel/test-http-parser-lazy-loaded.js), but that's perhaps even more unsafe/unstable than `process.binding`.
